webplugin/js/app/kommunicate-ui.js

```diff
@@ -10,9 +10,10 @@ KommunicateUI={
     welcomeMessageEnabled : false,
     leadCollectionEnabledOnWelcomeMessage:false,
     anonymousUser:false,
+    showClosedConversations: false,
```

Comment:
@akshatdhabi can we name it Resolved here also and any other new code added

Reply:
@devashishmamgain  done.